workflows/api/runTests.ps1

```diff
@@ -0,0 +1,16 @@
+# Copyright(c) 2025 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+dotnet restore --force
```

Comment:
Because we have a 10 minute timeout on each of the tests, we need to increase the timeout for overall running of the tests to at least 25mins. Look at the Spanner or Storage samples as they do somthing similar already. Thanks.